scripts/retrieve_cutout.py

```diff
@@ -56,20 +54,12 @@
 
     configure_logging(snakemake) # TODO Make logging compatible with progressbar (see PR #102)
 
-    if snakemake.config['tutorial']:
-        url = "https://zenodo.org/record/3518020/files/pypsa-eur-tutorial-cutouts.tar.xz"
-    else:
-        url = "https://zenodo.org/record/3517949/files/pypsa-eur-cutouts.tar.xz"
-
-    # Save location
-    tarball_fn = Path(f"{rootpath}/cutouts.tar.xz")
-
-    logger.info(f"Downloading cutouts from '{url}'.")
-    progress_retrieve(url, tarball_fn)
+    cutout = snakemake.wildcards.cutout
 
-    logger.info(f"Extracting cutouts.")
-    tarfile.open(tarball_fn).extractall(path=rootpath)
+    # url = f"https://zenodo.org/record/3517949/files/{cutout}.nc"
+    url = f"https://sandbox.zenodo.org/record/795060/files/{cutout}.nc"
```

Comment:
What say we move the URL to the config files as well?
I.e. explicit entries with each cutout:
In config.default.yaml:
url: https://sandbox.zenodo.org/record/795060/files/europe-2013-era5.nc
url: https://sandbox.zenodo.org/record/795060/files/europe-2013-sarah.nc

In config.tutorial.yaml
url: https://sandbox.zenodo.org/record/795060/files/europe-2013-era5-tutorial.nc